services/r2ctl/main/main.go

```diff
@@ -105,7 +107,13 @@ func main() {
 	r2ApiServer := server.NewServer(
 		listenAddr,
 		serverOpts,
-		r2.NewService(instrumentOpts, r2apiPrefix, authService, store),
+		r2.NewService(
+			instrumentOpts,
+			r2apiPrefix,
+			authService,
+			store,
+			clock.NewOptions().SetNowFn(time.Now),
```

Comment:
nit: by default the now fn in a clock options is `time.Now` so you don't need to set it explicitly.